src/boot/lib/mexpr.ml

```diff
@@ -1881,6 +1885,21 @@ and delta (apply : info -> tm -> tm -> tm) fi c v =
       TmSeq (fi, tms)
   | CreadLine, _ ->
       fail_constapp fi
+  | CreadBytes (Some TmRecord (_, r)), TmConst (_, CInt v) ->
```

Comment:
I'm having problems here, I don't know which type I'm supposed to define in `src/boot/lib/ast.ml` in order to get a TmRecord in here.